### ISSUE TITLE
No longer downloading in rest mode after updating to v0.52

### ISSUE THREAD

**Issue by Kordsevered** [blastrock/pkgj#317]
Version of firmware: 3.60 Henkaku
Version of PKGj: 0.52

## Steps to reproduce the issue

- update to v0.52 from previously released version
- either enter rest mode by pressing power button or allow system to enter rest mode passively.

Obtained result: after about 1 minute, whatever game is being downloaded automatically pauses and no other subsequent downloads in queue start. Once the system is brought out of rest mode the next download in queue starts leaving the previous file in a paused state.

Expected result: Fill queue>enter rest mode >all downloads complete by morning (note: I had just achieved this result the night before this posting. Only system change made was PKGj v0.52 vkp update)

-Not much else to say about the issue. No other details to give really. Downloads work fine still while system is active. 
-side note (possibly a whole other topic) 
I installed iTLS-enso v2.0 as newly advised by PKGj and it caused the system not to boot with a error message at the epilepsy splash. Fixed by booting without plugins and uninstalling iTLS from ur0/tai. 
I should note here also that the main download issue is unrelated to this. It started directly following the 0.52 update and persisted through and after the iTLS problem. I only add this additional issue as a sort of disclaimer since PKGj is currently recommending iTLS in app.

**Comment by TheRadziu** [comment]
bgdl code was untouched between 0.51 and 0.52, so latest pkgj update couldn't introduce any new bugs.

You mention itls-enso v2.0 while latest itls-enso at the time of 0.52 pkgj update was 3.0 (now 3.1), which might explain the additional issues you mention. Please try to do the same with latest itls-enso installed and also with itls-enso completly uninstalled, as rest mode download might be related to gleba's plugin.

**Comment by lanner** [comment]
Yes you have to update to ITLS 3.1 , i have install all yesterday for my case

**Comment by Kordsevered** [comment]
Update: 
My apologies for the greatly delayed response. I reviewed my plugins and I did indeed have the wrong itls-enso ver. I had just been running with it disabled until I caught your replies. All up to date now. Thank you, that was a silly oversight. However the background download issue seemingly fixed itself after about 9 days. I didnt change anything. No installs or uninstalls. It seemingly corrected itself just as mysteriously as it started. Also worth noting it wasnt anything simple like power cycling either. I regularly power cycle when making any changes in vita shell ect. No further issues since. Thank you for your assistance!
